camera_names: 
  - 'camera_front'
  - 'camera_right'
  - 'camera_left'